### PR TITLE
Zustand

### PR DESCRIPTION
**Pull request by desplodur**
Please take a look

In this feature i added Zustand into my application. 
Furthermore i did a small adjustment on the eslint script

### CODE REVIEW

File: src/components/QuestionComponent/index.js [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
 
-export default function QuestionComponent({question, addAnswer}) {
+export default function QuestionComponent({question}) {
 	const [toggle, setToggle] = useState(true);
+	const questions = useStore(state => state.questions);
+	const addNewAnswer = useStore(state => state.addNewAnswer);
+
+	const addAnswer = (event, id) => {
+		event.preventDefault();
+		const index = questions.findIndex(question => {
```
Comment (stefankorder) [inline src/components/QuestionComponent/index.js:16]: Better solution would be possible with map instead your solution from line 16 to 24

**Review by stefankorder — APPROVED** [review]
👍🏻

File: src/components/QuestionComponent/index.js (hunk 1 above)
Comment (desplodur) [reply src/components/QuestionComponent/index.js]: Done :)

**Review by stefankorder — APPROVED** [review]
👍🏻

**Pull request merged**